voice-client-javascript/assets/admin/statuses.private.js

```diff
@@ -39,31 +37,31 @@ async function checkEnvironmentInitialization(context) {
     ];
   } else {
     status.valid = true;
-    status.description = `Your application is initialized! View your [running application](../index.html)`;
+    status.description = `Your application is initialized! View your [running application](../browser.html)`;
```

Comment:
we have a helper utility you can use to generate the deployed url here https://github.com/twilio-labs/function-templates/blob/main/docs/static/v1/ce-helpers.js